src/transformers/models/mbart/tokenization_mbart.py

```diff
@@ -110,7 +110,7 @@ def __init__(
             cls_token=cls_token,
             pad_token=pad_token,
             mask_token=mask_token,
-            tokenizer_file=tokenizer_file,
+            tokenizer_file=None,
```

Comment:
I propose this change to avoid that the `tokenizer_file` argument is recorded in the tokenizer_config file